source/includes/code-examples/FindMany.cs

```diff
@@ -0,0 +1,98 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindMany
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Find multiple documents using builders
+        WriteLine("Finding documents with builders...:");
+        FindMultipleRestaurantsBuilderSync();
+
+        // Extra space for console readability 
+        WriteLine();
+
+        // Find multiple documents using LINQ
+        WriteLine("Finding documents with LINQ...:");
+        FindMultipleRestaurantsLINQSync();
+
+        WriteLine();
+
+        // Find All restaurants
+        WriteLine("Finding all documents...:");
+        FindAllRestaurantsSync();
+    }
+
+    private static void FindMultipleRestaurantsBuilderSync()
+    {
+        // start-find-builders-sync
+        var filter = Builders<Restaurant>.Filter
+            .Eq(r => r.Cuisine, "Pizza");
+
+        var restaurants = _restaurantsCollection.Find(filter).ToList();
+        // end-find-builders-sync
+
+        WriteLine("Number of documents found: " + restaurants.Count);
+    }
+
+    private static void FindMultipleRestaurantsLINQSync()
+    {
+        // start-find-linq-sync
+        var query = _restaurantsCollection.AsQueryable()
+            .Where(r => r.Cuisine == "Pizza").ToList();
+        // end-find-linq-sync
+
+        WriteLine("Number of documents found: " + query.Count);
+    }
+
+    private static void FindAllRestaurantsSync()
+    {
+        // start-find-all-sync
+        var restaurants = _restaurantsCollection.Find(Builders<BsonDocument>.Filter.Empty())
+            .ToList();
+        // end-find-all-sync
+        WriteLine("Number of documents found: " + restaurants.Count);
+    }
+
+    private static void Setup()
+    {
+        // This allows automapping of the camelCase database fields to our models. 
+        var camelCaseConvention = new ConventionPack { new CamelCaseElementNameConvention() };
+        ConventionRegistry.Register("CamelCase", camelCaseConvention, type => true);
+
+        // Establish the connection to MongoDB and get the restaurants database
+        var mongoClient = new MongoClient(_mongoConnectionString);
+        var restaurantsDatabase = mongoClient.GetDatabase("sample_restaurants");
+        _restaurantsCollection = restaurantsDatabase.GetCollection<Restaurant>("restaurants");
+    }
+}
+
+// start-model
+public class Restaurant
+{
+    public ObjectId Id { get; set; }
+
+    public string Name { get; set; }
+
+    [BsonElement("restaurant_id")]
```

Comment:
I'm not sure why do we define a BsonElement for a single field. I think, it should be either for all or nothing?

Reply:
I don't know definitively. @BorisDog or @rstam, how should we use [BsonElement] in this class?

Reply:
I know how we can use it :) , the question is why we define it only for one field? I would expect either defining it for all props or for no one

Reply:
Oh, sorry Dima! 

When I remove the attribute from that property, I get a runtime error, but when I add it to just that one property, it works. You're saying it's still good practice to add it everywhere, even if it isn't strictly required?

Reply:
this attribute just says that a db value in `restaurant_id` field will be mapped into `RestaurantId`. Exception you saw previously said that this mapping is not configured, but it's only because you already had `restaurant_id` in your collection. An empty collection won't give this issue

Reply:
note: I don't say we need to remove it, I simple mean we should choose a single approach